MantidQt/CustomInterfaces/src/Indirect/ConvFit.cpp

```diff
@@ -1319,14 +1288,14 @@ void ConvFit::singleFitComplete(bool error) {
     pref += "f" + QString::number(subIndex) + ".";
   }
 
-  if (fitTypeIndex == 1 || fitTypeIndex == 2) {
+  if (fitTypeIndex == 2) {
     functionName = "Lorentzian 1";
```

Comment:
Is there any reason why we can't move the assignment of `functionName` to "Lorentzian 1" to the second if block on line 1295? The first if block doesn't look like its needed anymore.